### PR TITLE
Hides extra map elements

### PR DESCRIPTION
**Pull request by DonovanAndrews300**
## **Description**
This PR removes the map scale on the bottom left of the property map and hides the map attribute on the bottom right of the map by default.  Addresses [(Issue#1022)](https://github.com/CodeForPhilly/clean-and-green-philly/issues/1022#issuecomment-2543404412) 

---

## **Steps to Test**

1. Navigate to the Property Map
2. You should not see the map scale on the bottom left corner of the map just the map legend
3. The attribution on the bottom right should be hidden when the application loads
---

## **Screenshots**
Screenshot from 2024-12-18 11-23-06

### CODE REVIEW

**Comment by nlebovits** [comment]
@CodeWritingCow can we get a PR review?

File: src/components/PropertyMap.tsx [hunk 1]
```diff
@@ -445,6 +444,14 @@ const PropertyMap: FC<PropertyMapProps> = ({
         }}
         onLoad={(e) => {
           setMap(e.target);
+          const attributionButton = document.querySelector(
+            '.maplibregl-ctrl-attrib-button'
+          );
+          if (attributionButton) {
+            attributionButton.click();
```
Comment (CodeWritingCow) [inline src/components/PropertyMap.tsx:451]: @DonovanAndrews300 TypeScript is throwing an error saying that `click` doesn't exist on `Element`

**Comment by CodeWritingCow** [comment]
Hi @DonovanAndrews300 I ran your PR's code locally on my computer, and the UI works as expected.

However, GitHub failed to deploy the PR to Vercel's preview environment. I suspect it's due to a minor TypeScript error. TS says `click` doesn't exist on `Element`:

Screen Shot 2024-12-27 at 4 12 00 PM

The fix should be pretty easy. If we just annotate `attributionButton`'s type as an HTMLElement, then the error goes away:

Screen Shot 2024-12-27 at 4 23 35 PM

File: src/components/PropertyMap.tsx (hunk 1 above)
Comment (CodeWritingCow) [inline src/components/PropertyMap.tsx:451]: TypeScript is throwing a type error saying `click` doesn't exist on `attributeButton`

**Review by CodeWritingCow — CHANGES_REQUESTED** [review]
Hi @DonovanAndrews300 can you add a type annotation for attributionButton? I'll then approve and merge your PR. Thanks!

**Review by CodeWritingCow — APPROVED** [review]
@DonovanAndrews300 PR looks good. Thanks!

**Pull request merged**